glacierscript.py

```diff
@@ -840,6 +855,7 @@ def withdraw_interactive():
     parser.add_argument(
         "-n", type=int, help="Number of total keys required in an m-of-n multisig address creation (default m-of-n = 1-of-2)", default=2)
     parser.add_argument('--testnet', type=int, help=argparse.SUPPRESS)
+    parser.add_argument("-q", "--qrdata", help="Data to be encoded into qr-code")
```

Comment:
What is the purpose and use case of this flag? And of this entire commit? Creating multiple withdrawals in one sitting?

Reply:
The multi withdrawals in one session is why I wrote the qr suffix increment. I put in this qr-code -q business to handle getting secure data out of q environment in a secure-ish way. This is important/needed for me because I don't want to use paper keys and plan on using veracrypted volumes/data/keys. An example of data to export would be the cold storage public address (post the initial script run). People using paper keys have it on their paper, but I won't and might need to re-acquire it for a new location. Then there is "why not just use qrencode"? I like the feedback checking the already-written qr function so that's why I introduced the CLI argument rather than planning on using qrencode independently (I could be wrong about the advantages here though). Thanks again for reviewing all of this.

Reply:
Hmm...still not sure I understand (the cold storage address is already QR-encoded as part of the create-deposit-data, why do you need to QR-encode it again later?), but it sounds like this is a niche use case that doesn't really belong in this global repo. Better not to have code in GlacierScript that is not documented and has no use case except for your special situation.

Reply:
I think that if more people elect to not use paper storage it will be less of a niche case, under the assumption that diligent QRs are not necessarily being kept (easier to store such information in keepass inside a veracrypt volume). I will go through the doc revisions once I have my code additions/proposals in respectable form. But I will respect your opinion and segregate this out. Thanks!

Reply:
Using electronic storage for keys is a major change to Glacier and its security model. IMO that doesn't belong in this primary Glacier repo, even as an option. (See principle 5 in the design document: Offer Limited Choice.)

I'd recommend you do your own fork of Glacier to support electronic storage of keys.

Reply:
Understood, will do - thanks for the response!